mathesar/templates/mathesar/login_base.html

```diff
@@ -112,41 +112,6 @@
     <img src="{% static 'images/red-logo-with-text.svg' %}" alt="Mathesar Logo" class="align-center"/>
   </div>
 
-  {% if live_demo_mode %}
-    <div class="unsupported-device">
-      <div class="warning-icon">⚠️</div>
-      <p class="title">{% translate "Unsupported Screen Size" %}</p>
-      <p>
-      {% translate "Mathesar is a spreadsheet-like application with a rich UI that does not yet function well on screens this small. Improved support for mobile devices is on our" %}
-        
-        <a href="https://mathesar.org/roadmap.html" target="_blank">
-          {% translate "roadmap" %}
-        </a>.
-      </p>
-      <p>
-      {% translate "You can still use this demo site, but some features may not work correctly. We encourage you to try Mathesar on a device with a larger screen." %}
-    </p>
-    </div>
-    <div class="tutorial align-center">
-      <div class="header">
-        <strong>{% translate "Live Demo Mode" %}</strong>
-      </div>
-      <div class="body">
-      {% translate "Mathesar's live demo is available to anyone to try out." %}
-        
-        {% if live_demo_username and live_demo_password %}
-          {% translate "Use the following credentials to login" %}:
-          <ul>
-            <li>{% translate "Username" %}: <strong>{{live_demo_username}}</strong></li>
-            <li>{% translate "Password" %}: <strong>{{live_demo_password}}</strong></li>
-          </ul>
-        {% endif %}
-
-        {% translate "Keep in mind that the data in the live demo is reset regularly." %}
-      </div>
-    </div>
-  {% endif %}
-
```

Comment:
We would have to figure out a different way to mention that the demo data is reset regularly to users accessing the demo site. Do we have an approach figured out on how to do that?

Reply:
I don't have an approach figured out yet. I figure let's get everything out for now, and then only add things back into the service if absolutely needed later. Couldn't we have a custom front page served by the demo microservice? It could pull the needed assets from the "real service". Not sure, though.

Reply:
I think having a custom front page should be fine.

However, we currently show a banner on the app while someone is viewing the demo. This PR removes that banner as well. I'm cool with removing it and adding it later based on requirements. I do think this needs to be tracked in a separate issue.